concordium-consensus/src/Concordium/Skov/Update.hs

```diff
@@ -1,22 +1,21 @@
 {-# LANGUAGE
+    UndecidableInstances,
     ScopedTypeVariables,
-    TypeFamilies,
-    ViewPatterns #-}
+    ViewPatterns,
+    TemplateHaskell,
+    StandaloneDeriving,
+    DerivingVia #-}
```

Comment:
Are you sure these additional extensions are needed? Unless I'm missing something, you are not declaring new types and adding instances, so at least UndecidableInstances and DerivingVia should not be necessary.

Reply:
You're right. Forgot to remove them when I moved some code around.